app/helpers/application_helper.rb

```diff
@@ -15,7 +15,7 @@ def generate_history_text url
       puts "#"*20
       puts e.message
       puts "#"*20
-      return url
+      return "/" # default to home page.
```

Comment:
I updated 'return "/"' because of the goofy mouseover "title" that was being displayed after a user had just logged in with Shibboleth, but mouseover titles aren't that important so I can take it out.